src/client/components/baseComponents/PlayerStatsTable.ts

```diff
@@ -0,0 +1,199 @@
+import { css, html, LitElement } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import { PlayerStats } from "../../../core/StatsSchemas";
+
+@customElement("player-stats-table")
+export class PlayerStatsTable extends LitElement {
+  static styles = css`
+    .table-container {
+      margin-top: 1rem;
+      width: 100%;
+      max-width: 28rem;
+    }
+    table {
+      width: 100%;
+      font-size: 0.95rem;
+      color: #ccc;
+      border-collapse: collapse;
+    }
+    th,
+    td {
+      padding: 0.25rem 0.5rem;
+      text-align: center;
+    }
+    th {
+      color: #bbb;
+      font-weight: 600;
+    }
+    .section-title {
+      color: #888;
+      font-size: 1rem;
+      font-weight: bold;
+      margin-bottom: 0.5rem;
+    }
+  `;
+
+  @property({ type: Object }) stats: PlayerStats;
+
+  private getBuildingName(key: string): string {
+    const names: Record<string, string> = {
+      // buildings
+      city: "City",
+      port: "Port",
+      defp: "Defense",
+      saml: "SAM",
+      silo: "Missile Silo",
+      // boats
+      trade: "Trade Ship",
+      trans: "Transport Ship",
+      wshp: "Warship",
+      // bombs
+      abomb: "Atom Bomb",
+      hbomb: "Hydrogen Bomb",
+      mirv: "MIRV",
+    };
+    return names[key] ?? key;
+  }
+
+  render() {
+    const stats = this.stats;
+
+    const buildingKeys = ["city", "port", "defp", "saml", "silo"];
+    const boatKeys = ["trade", "trans", "wshp"];
+    const bombKeys = ["abomb", "hbomb", "mirv"];
+
+    return html`
+      <div class="table-container">
+        <div class="section-title">🏗️ Building Statistics</div>
+        <table>
+          <thead>
+            <tr>
+              <th class="text-left">Building</th>
+              <th>Built</th>
+              <th>Destroyed</th>
+              <th>Captured</th>
+              <th>Lost</th>
+            </tr>
+          </thead>
+          <tbody>
+            ${buildingKeys.map((key) => {
+              const built = Number(stats?.units?.[key]?.[0] ?? 0);
+              const destroyed = Number(stats?.units?.[key]?.[1] ?? 0);
+              const captured = Number(stats?.units?.[key]?.[2] ?? 0);
+              const lost = Number(stats?.units?.[key]?.[3] ?? 0);
+              return html`
+                <tr>
+                  <td>${this.getBuildingName(key)}</td>
+                  <td>${built}</td>
```

Comment:
```suggestion
                  <td>${renderNumber(built)}</td>
```

same comment for other numbers in this file